src/CMakeLists.txt

```
@@ -666,9 +663,10 @@ set(INCLUDE_FILES
         drivers/TwiMaster.h
         heartratetask/HeartRateTask.h
         components/heartrate/Ppg.h
-        components/heartrate/Biquad.h
-        components/heartrate/Ptagc.h
         components/heartrate/HeartRateController.h
+	libs/arduinoFFT-develop/src/arduinoFFT.h
+	libs/arduinoFFT-develop/src/defs.h
+	libs/arduinoFFT-develop/src/types.h
```

The indentation here uses a tab, rather than spaces.

```suggestion
        libs/arduinoFFT-develop/src/arduinoFFT.h
        libs/arduinoFFT-develop/src/defs.h
        libs/arduinoFFT-develop/src/types.h
```

---

@FintasticMan Thanks! I will fix.